lib/glimmer/tk/toplevel_proxy.rb

```diff
@@ -32,19 +32,43 @@ class ToplevelProxy < WidgetProxy
       DEFAULT_HEIGHT = 0
       
       attr_reader :tk
-      attr_accessor :escapable
+      attr_accessor :escapable, :modal, :centered
       alias escapable? escapable
-      
+      alias modal? modal
+      alias centered? centered
+
       def post_add_content
-        center_within_screen unless @x || @y
+        center_within_screen if centered?
```

Comment:
Centering was done as a smart default in the past unless the `x` and `y` attributes were specified. I think it is important to maintain this smart default without requiring the user to specify `centered true`. 

But, you can support `centered false` if you want to prevent centering when the user neither wants the default and nor wants to specify the x and y manually.